go/ql/test/library-tests/semmle/go/dataflow/FlowSteps/LocalFlowStep.expected

```diff
@@ -127,12 +127,10 @@
 | main.go:64:7:64:18 | call to min | main.go:64:2:64:2 | definition of a |
 | main.go:64:11:64:11 | x | main.go:64:7:64:18 | call to min |
 | main.go:64:14:64:14 | y | main.go:64:7:64:18 | call to min |
-| main.go:64:17:64:17 | z | main.go:64:7:64:18 | call to min |
```

Comment:
Lost results due to min and max being variadic functions

Reply:
This has been fixed now.